### ISSUE TITLE
Add the Notification node (ui-toast) that is present in Dashboard 1

### ISSUE THREAD

**Issue by colinl** [flowforge/flowforge-nr-dashboard#149]
### Description

D1 includes a Notification node (ui_toast). It appears to be missing from D2

### Have you provided an initial effort estimate for this issue?

I am no FlowForge team member

**Comment by joepavitt** [comment]
About to raise a PR for this, but have in the process, made a conscious decision to remove the "Dialog" functionality from ui-toast (now ui-notification).

Instead, I will be adding a new widget ui-dialog, which will (if all goes to plan) allow for diverse input in a Dialog Modal, beyond what D1.0 would allow (single input)

**Comment by flinthamm** [comment]
First off, thank you for this amazing project and I look forward to ongoing developments. 
Sorry to ask such a trivial question but could someone give an example of how to display a ui-notification as depicted in the screenshot i.e. with a bold title and message directly underneath? I found examples of dashboard 1, where it appeared sending a msg.topic would set the title and the msg.payload the message but this did not work (no title, just message). I've tried sending basic html formatting e.g.
```
<p><string>Generated Notification</strong><br />This is custom HTML...</p>
<p><string>Generated Notification</strong></p><p>This is custom HTML...</p>
```
Only the title is displayed but perhaps I need to introduce inline CSS?
